tests/test_hf_api.py

```diff
@@ -1116,14 +1084,12 @@ def test_model_info(self):
         )
         self.assertIsInstance(model_info, ModelInfo)
 
-    @with_production_testing
-    def test_list_private_models(self):
+    def test_list_private_datasets(self):
```

Comment:
Not sure why this was tested against production?

Reply:
cc @LysandreJik